spec/policies/membership_application_policy_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+
+describe MembershipApplicationPolicy do
+
+  let(:application_owner) { create(:user, email: 'user_1@random.com') }
+  let(:admin) { create(:user, email: 'admin@sgf.com', admin: true) }
+
+  let(:application) { create(:membership_application,
+                             user: application_owner) }
+
+  subject { MembershipApplicationPolicy.new(application_owner, application) }
```

Comment:
You are using `described_class` in other parts of the test. Why `MembershipApplicationPolicy` here?